### PR TITLE
Add Close() for writer and reader.

### PR DESCRIPTION
**Pull request by alexuserid**
Add Close() for reader and writer. Close() for writer produces EOF and triggers loop break here https://github.com/BurkovBA/go-kafka-adapter/blob/2c33b59977de28fc74eb5e8203675f122d3cb56a/kafka_store.go#L265
It's common approach to close reader too when it's not needed anymore.

### CODE REVIEW

File: kafka_store.go [hunk 1]
```diff
@@ -42,8 +42,8 @@ func (handler *KafkaStoreConsumerGroupHandler) isAtHighwatermark(session sarama.
 
 	fmt.Printf("highWatermark = %d, offset = %d\n", highWatermark, offset)
 
-	// if we've reached highWatermark, return
-	if offset >= highWatermark {
+	// if we've reached highWatermark or there's no messages in kafka at all, return
```
Comment (BurkovBA) [inline kafka_store.go]: Tiny grammar fix: "there's" -> "there are"

Comment (BurkovBA) [reply kafka_store.go]: @alexuserid Thank you for your PR! 

Please fix a tiny grammar error in a comment and I'll merge it. By the way, do I know you in real life?)

Comment (alexuserid) [reply kafka_store.go]: Fixed.
No, we haven't met, but I'm working with Pavel Dolgov and Boris Nagaev, I guess you know them).

**Comment by BurkovBA** [comment]
Thank you, @alexuserid.

**Pull request merged**